pymc3/distributions/continuous.py

```diff
@@ -194,7 +210,7 @@ def logp(self, value):
                      tau > 0, sd > 0)
 
 
-class HalfNormal(PositiveContinuous):
+class HalfNormal(PositiveUnivariateContinuous):
```

Comment:
This might be a case where we want to start using multiple inheritance, e.g. `HalfNormal(Positive, Univariate, Continuous)`